debian/changelog

```diff
@@ -1,3 +1,21 @@
+opx-base-model (3.151.0+opx3) unstable; urgency=medium
```

Comment:
@hpersh Do you want to merge all the 3 changes into one version bump (3.151.0+opx1)?